doc/requirements.rst

```diff
@@ -135,32 +141,36 @@ Motor 2.5 deprecated support for Python 3.5.
 
 Motor 3.0 dropped support for Pythons older than 3.7.
 
+Motor 3.1.1 added support for Python 3.11.
+
 +-------------------------------------------------------------------------------------+
 |                   Python Version                                                    |
 +=====================+=====+=====+=====+=======+=======+=====+=====+=====+=====+=====+
```

Comment:
the `+======` and `+------` lines need to be extended for the new 3.11 column: https://github.com/mongodb/motor/actions/runs/3322825314/jobs/5492384749